### DISCUSSION TITLE
Update map

### DISCUSSION THREAD

**Discussion by Ninovui** [Endebert/squadmc#148] [Q&A]
Hi !

I just went out of a Squad session and I noticed that some maps doesn't match the graphics of the tactical map of the game. It's the case with Gorodok, where the map isn't the same that in game. I have to use the minimalist squad calculator because it is impossible to use the squadmc calculator correctly. Is it possible to make a tour of the whole layers and update the map ? It would be very helpful. Thanks !

**Comment by sh4rkman** [comment]
Several people [have opened pull requested](https://github.com/Endebert/squadmc-maps/pulls) fixing this problem, but it looks like @Endebert isn't active anymore and this issues are staying for now.

You can either stick to my minimalist calculator or use https://squadmortar.xyz if you want a clickable interface while we wait for Endebert return

**Comment by SatanicGeek** [comment]
Any update on this ?
I wasn't able to find a blog or something else where @Endebert could have said something about the project.
